src/domain/reservation/manage/information/ManageReservationsInfoModal.js

```diff
@@ -0,0 +1,78 @@
+import React from 'react';
+import {
+  Modal, Row, Col, Button
+} from 'react-bootstrap';
+import PropTypes from 'prop-types';
+import get from 'lodash/get';
+
+import ManageReservationsStatus from '../status/ManageReservationsStatus';
+import injectT from '../../../../../app/i18n/injectT';
+import { getDateAndTime } from '../list/ManageReservationsList';
+
+const ManageReservationsInfoModal = ({
```

Comment:
The component name and folder name doesn't match (information vs info). I think that the component should be called `ManageReservationsReservationModal` and folder should be named as `reservationModal` or `ManageReservationsReservationInfoModal` and `reservationInfoModal`.

Reply:
We have `reservation` folder already,  and this file was under it so i guess `ManageReservationsInformationModal` will fit? folder will be `informationModal`
or simply just `ManageReservationsModal` and folder name is `modal`

We wont have any other modal this in dir

Reply:
Well, it's just a matter of semantics. `ManageReservations` is the manage reservations feature/page, and that modal is a modal for showing/editing reservation information. It's not for showing "manage reservations" information.

Reply:
What does `ManageReservationsInformationModal` mean?

Reply:
Then bring ti out of `manage` folder and do a new folder under `resource` name `informationModal` ?

Reply:
Sounds better, and component name would be `ReservationInformationModal`. Or maybe just `modal/ReservationModal.js`?

Reply:
`modal` sound do better